src/pages/registration/resource_type_tab/components/RevisionField.tsx

```diff
@@ -0,0 +1,35 @@
+import { useTranslation } from 'react-i18next';
+import { Field, FieldProps, useFormikContext } from 'formik';
+import { ResourceFieldNames } from '../../../../types/publicationFieldNames';
+import { Checkbox, FormControlLabel, InputProps } from '@mui/material';
+import { BookRegistration, Revision } from '../../../../types/publication_types/bookRegistration.types';
+
+export const RevisionField = () => {
+  const { setFieldValue } = useFormikContext<BookRegistration>();
+  const { t } = useTranslation();
+
+  const handleRevisionChange = (checked: boolean, field: InputProps) => {
+    setFieldValue(ResourceFieldNames.Revision, checked ? Revision.REVISED : Revision.UNREVISED);
+  };
+
+  return (
+    <Field name={ResourceFieldNames.Revision}>
+      {({ field }: FieldProps) => (
+        <FormControlLabel
+          label={t('registration.is_revision')}
```

Comment:
sikkert bare spørsmål om tid før Eirik skal teste dette også ja, som du nevner :)
```suggestion
          data-testid={dataTestId.registrationWizard.resourceType.revisionField}
          label={t('registration.is_revision')}
```